redaxo/src/core/pages/profile.php

```diff
@@ -161,6 +164,17 @@
 $fragment->setVar('elements', $formElements, false);
 $content .= $fragment->parse('core/form/form.php');
 
+$formElements = [];
+$n = [];
+$n['label'] = '<label for="rex-id-minibar">' . rex_i18n::msg('user_minibar') . '</label>';
+$n['field'] = '<input type="checkbox" id="rex-id-minibar" name="minibar" value="1"' . ($userminibar == 1 ? ' checked="checked"' : ''). ' />';
+$n['note'] = rex_i18n::msg('user_minibar_note');
```

Comment:
Wenn der User keine Berechtigung für die Minibar hat, sollte man ihm diese Einstellung gar nicht erst geben, oder?

Reply:
> Wenn der User keine Berechtigung für die Minibar hat

Es gibt bisher kein Recht dafür. Der bisherige Gedanke war, das der User selbst entscheiden kann ob er die Minibar sieht oder nicht. Ob er aber Zugriff auf die einzelnen Elemente in der Minibar bekommt, entscheidet dann wieder der Programmierer der sich mit einem neuen Element in die Minibar einklinkt.